velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -378,14 +390,29 @@ class PlanBuilder {
   /// @param step Aggregation step: partial, final, intermediate or single.
   /// @param ignoreNullKeys Boolean indicating whether to skip input rows where
   /// one of the grouping keys is null.
+  /// @param globalGroupingSets groupId values corresponding to global
+  /// aggregations from an underlying GroupId node used to conpute GroupingSets.
+  /// e.g. If the source GroupId is used for grouping sets {{orderkey}, {}}
```

Comment:
If it is required that source node is a GroupId, can we not automatically derive the necessary information without asking the user to specify it explicitly?

Reply:
I was trying to keep the API flexible. But that can be done.